<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RcmdWidget</class>
 <widget class="QDialog" name="RcmdWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>701</width>
    <height>388</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(30, 30, 30);
color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>291</width>
     <height>311</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
background: gray;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <widget class="QLabel" name="image1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>111</width>
      <height>161</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>맑은 고딕</family>
      <pointsize>9</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
font: 9pt &quot;맑은 고딕&quot;;;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="text">
     <string>선수사진</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QWidget" name="formLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>101</width>
      <height>111</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <item row="2" column="0">
      <widget class="QLabel" name="ss_13">
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 0, 0);
font: 12pt &quot;휴먼옛체&quot;;</string>
       </property>
       <property name="text">
        <string>스탯3</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="ss_11">
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 0, 0);
font: 12pt &quot;휴먼옛체&quot;;</string>
       </property>
       <property name="text">
        <string>스탯1</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="ss_12">
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 0, 0);
font: 12pt &quot;휴먼옛체&quot;;</string>
       </property>
       <property name="text">
        <string>스탯2</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="ss_14">
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 0, 0);
font: 12pt &quot;휴먼옛체&quot;;</string>
       </property>
       <property name="text">
        <string>스탯4</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="stat_11">
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);
font: 12pt &quot;휴먼옛체&quot;;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="stat_12">
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);
font: 12pt &quot;휴먼옛체&quot;;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="stat_13">
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);
font: 12pt &quot;휴먼옛체&quot;;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLabel" name="stat_14">
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);
font: 12pt &quot;휴먼옛체&quot;;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="formLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>190</y>
      <width>111</width>
      <height>112</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout_2">
     <item row="1" column="0">
      <widget class="QLabel" name="ss_16">
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 0, 0);
font: 12pt &quot;휴먼옛체&quot;;</string>
       </property>
       <property name="text">
        <string>스탯6</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="ss_15">
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 0, 0);
font: 12pt &quot;휴먼옛체&quot;;</string>
       </property>
       <property name="text">
        <string>스탯5</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="ss_17">
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 0, 0);
font: 12pt &quot;휴먼옛체&quot;;</string>
       </property>
       <property name="text">
        <string>스탯7</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="ss_18">
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 0, 0);
font: 12pt &quot;휴먼옛체&quot;;</string>
       </property>
       <property name="text">
        <string>스탯8</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLabel" name="stat_18">
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);
font: 12pt &quot;휴먼옛체&quot;;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="stat_17">
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);
font: 12pt &quot;휴먼옛체&quot;;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="stat_16">
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);
font: 12pt &quot;휴먼옛체&quot;;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="stat_15">
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);
font: 12pt &quot;휴먼옛체&quot;;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="formLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>15</y>
      <width>141</width>
      <height>151</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout_3">
     <item row="0" column="0">
      <widget class="QLabel" name="label_4">
       <property name="styleSheet">
        <string notr="true">font: 9pt &quot;맑은 고딕&quot;;
color: rgb(0, 0, 0);</string>
       </property>
       <property name="text">
        <string>이름 :</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="oName">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_6">
       <property name="styleSheet">
        <string notr="true">font: 9pt &quot;맑은 고딕&quot;;
color: rgb(0, 0, 0);</string>
       </property>
       <property name="text">
        <string>포지션 :</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="oPosition">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_11">
       <property name="styleSheet">
        <string notr="true">font: 9pt &quot;맑은 고딕&quot;;
color: rgb(0, 0, 0);</string>
       </property>
       <property name="text">
        <string>나이 :</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="oAge">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_13">
       <property name="styleSheet">
        <string notr="true">font: 10t &quot;맑은 고딕&quot;;
color: rgb(0, 0, 0);</string>
       </property>
       <property name="text">
        <string>몸값 :</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLabel" name="oValue">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_14">
       <property name="styleSheet">
        <string notr="true">font: 10t &quot;맑은 고딕&quot;;
color: rgb(0, 0, 0);</string>
       </property>
       <property name="text">
        <string>키 : </string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLabel" name="oH">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_15">
       <property name="styleSheet">
        <string notr="true">font: 10t &quot;맑은 고딕&quot;;
color: rgb(0, 0, 0);</string>
       </property>
       <property name="text">
        <string>몸무게 :</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLabel" name="oW">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>281</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>190</y>
      <width>20</width>
      <height>111</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_3">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>10</y>
      <width>20</width>
      <height>161</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frame_2">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>10</y>
     <width>291</width>
     <height>311</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
background: gray;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <widget class="QLabel" name="image2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>111</width>
      <height>161</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>맑은 고딕</family>
      <pointsize>9</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);font: 9pt &quot;맑은 고딕&quot;;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="text">
     <string>선수사진</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QWidget" name="formLayoutWidget_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>101</width>
      <height>112</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout_7">
     <item row="2" column="0">
      <widget class="QLabel" name="ss_23">
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 0, 0);
font: 12pt &quot;휴먼옛체&quot;;</string>
       </property>
       <property name="text">
        <string>스탯3</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="ss_21">
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 0, 0);
font: 12pt &quot;휴먼옛체&quot;;</string>
       </property>
       <property name="text">
        <string>스탯1</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="ss_22">
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 0, 0);
font: 12pt &quot;휴먼옛체&quot;;</string>
       </property>
       <property name="text">
        <string>스탯2</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="ss_24">
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 0, 0);
font: 12pt &quot;휴먼옛체&quot;;</string>
       </property>
       <property name="text">
        <string>스탯4</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="stat_21">
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);
font: 12pt &quot;휴먼옛체&quot;;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="stat_22">
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);
font: 12pt &quot;휴먼옛체&quot;;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="stat_23">
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);
font: 12pt &quot;휴먼옛체&quot;;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLabel" name="stat_24">
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);
font: 12pt &quot;휴먼옛체&quot;;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="formLayoutWidget_8">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>190</y>
      <width>111</width>
      <height>112</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout_8">
     <item row="1" column="0">
      <widget class="QLabel" name="ss_26">
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 0, 0);
font: 12pt &quot;휴먼옛체&quot;;</string>
       </property>
       <property name="text">
        <string>스탯6</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="ss_25">
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 0, 0);
font: 12pt &quot;휴먼옛체&quot;;</string>
       </property>
       <property name="text">
        <string>스탯5</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="ss_27">
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 0, 0);
font: 12pt &quot;휴먼옛체&quot;;</string>
       </property>
       <property name="text">
        <string>스탯7</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="ss_28">
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 0, 0);
font: 12pt &quot;휴먼옛체&quot;;</string>
       </property>
       <property name="text">
        <string>스탯8</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLabel" name="stat_28">
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);
font: 12pt &quot;휴먼옛체&quot;;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="stat_27">
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);
font: 12pt &quot;휴먼옛체&quot;;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="stat_26">
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);
font: 12pt &quot;휴먼옛체&quot;;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="stat_25">
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);
font: 12pt &quot;휴먼옛체&quot;;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="formLayoutWidget_9">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>10</y>
      <width>141</width>
      <height>161</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout_9">
     <item row="1" column="0">
      <widget class="QLabel" name="label_49">
       <property name="styleSheet">
        <string notr="true">font: 10t &quot;맑은 고딕&quot;;
color: rgb(0, 0, 0);</string>
       </property>
       <property name="text">
        <string>이름 :</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="rName">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_51">
       <property name="styleSheet">
        <string notr="true">font: 10t &quot;맑은 고딕&quot;;
color: rgb(0, 0, 0);</string>
       </property>
       <property name="text">
        <string>포지션 :</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="rPosition">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_52">
       <property name="styleSheet">
        <string notr="true">font: 10t &quot;맑은 고딕&quot;;
color: rgb(0, 0, 0);</string>
       </property>
       <property name="text">
        <string>나이 :</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLabel" name="rAge">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_54">
       <property name="styleSheet">
        <string notr="true">font: 10t &quot;맑은 고딕&quot;;
color: rgb(0, 0, 0);</string>
       </property>
       <property name="text">
        <string>몸값 :</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLabel" name="rValue">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_17">
       <property name="styleSheet">
        <string notr="true">font: 10t &quot;맑은 고딕&quot;;
color: rgb(0, 0, 0);</string>
       </property>
       <property name="text">
        <string>키 : </string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLabel" name="rH">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_16">
       <property name="styleSheet">
        <string notr="true">font: 10t &quot;맑은 고딕&quot;;
color: rgb(0, 0, 0);</string>
       </property>
       <property name="text">
        <string>몸무게 :</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QLabel" name="rW">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="Line" name="line_4">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>10</y>
      <width>20</width>
      <height>161</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>271</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_6">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>190</y>
      <width>20</width>
      <height>111</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton_7">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>350</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(166, 209, 255);
color: rgb(0, 0, 0);


font: 12pt &quot;맑은 고딕&quot;;

</string>
   </property>
   <property name="text">
    <string>닫기</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>340</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;맑은 고딕&quot;;</string>
   </property>
   <property name="text">
    <string>추천선수</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_55">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>340</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;맑은 고딕&quot;;</string>
   </property>
   <property name="text">
    <string>보유선수</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>350</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 11pt &quot;맑은 고딕&quot;;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>관련영상</string>
   </property>
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/icon/icon/youtube (1).png</normaloff>:/icon/icon/youtube (1).png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>25</width>
     <height>25</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>130</y>
     <width>61</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="image.qrc">:/icon/icon/match (2).png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="image.qrc"/>
 </resources>
 <connections/>
</ui>
